libs/entitlement/build.gradle

```diff
@@ -9,12 +9,25 @@
 apply plugin: 'elasticsearch.build'
 apply plugin: 'elasticsearch.publish'
 
+apply plugin: 'elasticsearch.embedded-providers'
+
+embeddedProviders {
+  impl 'entitlement', project(':libs:entitlement:asm-provider')
+}
+
+configurations {
+  entitlementBridge
+}
+
 dependencies {
   compileOnly project(':libs:core') // For @SuppressForbidden
+  compileOnly project(':libs:logging')
   compileOnly project(":libs:x-content") // for parsing policy files
-  compileOnly project(':server') // To access the main server module for special permission checks
   compileOnly project(':libs:entitlement:bridge')
-  testImplementation project(":test:framework")
+  testImplementation(project(":test:framework")) {
+    exclude group: 'org.elasticsearch', module: 'entitlement'
+  }
+  entitlementBridge project(':libs:entitlement:bridge')
```

Comment:
I'm not seeing where this configuration is actually used. What am I missing?

Reply:
I think you're right.

Reply:
I've looked into it a little and that would be a nontrivial change. There are at least two other places affected:
- `initPhase1` installs a dummy `SecurityManager`
- `AllPermissionsCheck` checks that the `SecurityManager` is working

Replacing these is not difficult but it's not trivial either. Perhaps this can wait for another PR?